docs/scos/dev/feature-walkthroughs/202204.0/configurable-product-feature-walkthrough/configurable-product-flow-chart.md

```diff
@@ -0,0 +1,198 @@
+|---|---|---|
+| `sourceType` | `pdp`, `cart-page` | Defines the page type where the configurator request is started. Resolves the `backUrl`, when the configurator response is processed. |
+| `SKU` | `some_sku` |   |
+| `Quantity` | `2` |   |
+
+The following table contains request parameters which the plugin adds to the request:
+
+| PARAMETER | VALUE |
+|---|---|
+| `idCustomer` | `DE-1` |
+| `localeName` | `de_DE` |
+| `storeName` | `DE` |
+| `currencyCode` | `EUR` |
+| `priceMode` | `NET_MODE` or `GROSS_MODE` |
+| `backUrl` | `https://SOME_URL` |
+
+### Phase 2
+
+The customer clicks the configuration button, and the request is redirected to the gateway page with a given product configuration using a `GET` method.
+
+### Phase 3
+
+1. The configuration data is read from the session or storage for the given SKU.
+2. A plugin that handles the request is selected based on the configurator type.
+3. The plugin expands the request with additional necessary data:
+  - store, locale, currency, customer, price mode.
+  - `backUrl`—based on the referer header.
+4. The plugin generates the URL that points to the configurator page together with all the necessary data.
+
+### Phase 4
+
+1. The request comes from the gateway to the configurator `index.php` (phase 4.1).
+2. The configurator handles the request by creating a new session and saving the request's data there (phase 4.2).
+3. The configurator responds with a redirect URL that contains what session ID needs to be picked up (phase 4.2).
+4. The request comes from the gateway to the configurator `index.php` to pick up a session (phase 4.3).
+5. Redirects the customer to the configurator page by a GET request using a secured connection.
+
+### Phase 5
+
+1. You are on the configurator page.
+2. The configurator page is rendered based on the data in the session.
+3. The customer does the configuration.
+4. Submits the configuration form.
+5. The configurator redirects the customer to the gateway page with configuration data.
+
+| PARAMETER | VALUE | COMMENT |
+|---|---|---|
+| `ProductConfigurationInstance.price` | `123` | Sensitive data. |
+| `ProductConfigurationInstance.isComplete` | `true` | Sensitive data. |
+| `ProductConfigurationInstance.availableQuantity` | `2` |   |
+| `ProductConfigurationInstance.displayDescription` | Some text of JSON blob—for example, `{"color"=>"red", "weight"=> 100}` |   |
+| `ProductConfigurationInstance.configuration` | `{"color"=>"red", "weight"=> 100}`   | Sensitive data. |
+| `idCustomer` | `DE-1` |   |
+| `sourceType` | `pdp`, `cart-page` |   |
+| `SKU`  | `some_sku` |   |
+| `timestamp` | `1231313123123` |   |
+| `CheckSum` | It’s an encrypted value of the `CheckSum`. It should be based on the all requested parameters and should have the same order for decryption. |   |
+
+### Phase 6
+
+1. The user finishes the configuration and clicks a button which creates an AJAX POST request with the data to the configurator page (self). (phase 6.1)
```

Comment:
We have the numbered list here. Do we still need to list subphases like _phase 6.1_ ,etc)  in parentheses?

Reply:
No, I don't think so.